Providers/Extras/Scripts/omsconfig_logrotate.conf

```diff
@@ -4,3 +4,10 @@
     weekly
```

Comment:
I would recommend specifying a limit size in this configuration as well. Currently, I've almost never seen omsconfig.log get rotated.

Reply:
omsconfig.log gets rotated. I will add size limit here. 

---
In reply to: [139852901](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/327#discussion_r139852901) [](ancestors = 139852901)